### PR TITLE
dockerfile debian based

### PR DESCRIPTION
**Pull request by AlvaroVega**
- [x] Tested

### CODE REVIEW

**Comment by AlvaroVega** [comment]
equivalent with centos8 https://github.com/telefonicaid/orchestrator/pull/321

File: ChangeLog [hunk 1]
```diff
@@ -2,6 +2,7 @@ CHANGES
 =======
 ADD: config env vars for PEP and IOTAGENT user name
 
+Move Dockerfile base image from centos7.9.2009 to Debian 11-slim
```
Comment (fgalan) [inline ChangeLog:5]:
```suggestion
Move Dockerfile base image from centos7.9.2009 to Debian 11.2-slim
```

(Debian 11 is fine, but explaining 11.2 we are more precise)

File: ChangeLog [hunk 2]
```diff
@@ -2,6 +2,7 @@ CHANGES
 =======
 ADD: config env vars for PEP and IOTAGENT user name
 
+Move Dockerfile base image from centos7.9.2009 to Debian 11.2-slim
 Upgrade: request from 2.25.1 to 2.27.1
 Upgrade: pymongo from 3.7.2 to 3.12.3
```
Comment (fgalan) [inline ChangeLog:7]: Maybe even better:

```suggestion
ADD: config env vars for PEP and IOTAGENT user name
UPGRADE: Dockerfile base image from centos7.9.2009 to Debian 11.2-slim
UPGRADE: request from 2.25.1 to 2.27.1
UPGRADE: pymongo from 3.7.2 to 3.12.3
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**